src/BizHawk.Client.EmuHawk/BizHawk.Client.EmuHawk.csproj

```diff
@@ -9,10 +9,10 @@
 		<ApplicationManifest>app.manifest</ApplicationManifest>
 		<Nullable>disable</Nullable>
 		<ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
+		<UseWindowsForms>true</UseWindowsForms>
 	</PropertyGroup>
 	<ItemGroup>
 		<PackageReference Include="System.Net.Http" />
-		<Reference Include="System.Windows.Forms" />

```

Comment:
Revert unrelated changes.

Reply:
I'll extract this change to master for all projects